esp/esp/program/forms.py

```diff
@@ -62,23 +62,46 @@ class ProgramCreationForm(BetterModelForm):
     student_reg_end   = forms.DateTimeField(widget = DateTimeWidget())
     base_cost         = forms.IntegerField(label = 'Cost of Program Admission $', min_value = 0 )
     sibling_discount  = forms.DecimalField(max_digits=9, decimal_places=2, required=False, initial=None,
-                                                help_text="The amount of the sibling discount. Leave blank if you don't use sibling discounts.")
+                                           help_text="The amount of the sibling discount. Leave blank if you don't use sibling discounts.")
     program_type      = forms.CharField(label = "Program Type", help_text='e.g. Splash or Cascade')
-    program_modules   = forms.MultipleChoiceField(
-                          choices=[],
-                          label='Program Modules',
-                          widget=forms.SelectMultiple(attrs={'class': 'input-xxlarge'}),
-                          help_text=Program.program_modules.field.help_text)
+    program_module_questions   = forms.MultipleChoiceField(choices=[],
+                                                           label='Program Modules',
+                                                           widget=forms.CheckboxSelectMultiple(attrs={'class': 'input-xxlarge'}),
+                                                           help_text=Program.program_modules.field.help_text,
+                                                           required=False)
+
+    program_modules   = forms.MultipleChoiceField(choices=[], widget = forms.SelectMultiple(attrs={'class': 'hidden-field'}), required=False)
 
     def __init__(self, *args, **kwargs):
         """ Used to update ChoiceFields with the current modules. """
+        # These modules are the "choosable" ones that admins will usually want to choose to select or exclude (i.e. not automatically include or exclude)
+        self.program_module_question_ids = OrderedDict({'Will you charge for the program?': [x.id for x in ProgramModule.objects.filter(admin_title__in=['Accounting', 'Financial Aid Application', 'Easily Approve Financial Aid Requests'])],
```

Comment:
Could we pull this information from the "financial details" section since it already asks for cost of program admission there? And could we maybe move the credit card, extra costs, donations questions to the financial details section?

Reply:
Yes, but it would be a lot of extra code to make an exception from the loop. @willgearty thoughts on hard-coding this as a separate question `<input>` item that the JS ties into the module list?

Reply:
I say we stick with the for loop for now. We can modify this later after SR12.